### ISSUE TITLE
Delete Jira `TEST` project

### ISSUE THREAD

**Issue by NotMyFault** [jenkins-infra/helpdesk#4005]
### Service(s)

Jira

### Summary

https://issues.jenkins.io/projects/TEST/summary is unused since 2020, nor has it been used before.

**Comment by timja** [comment]
it is the test project for the IRC bot (which isn't used much but isn't retired), is there a need to delete it?

**Comment by dduportal** [comment]
> test project for the IRC bot

I've updated the project description to describe this use case (and document it somehow)

@NotMyFault any objection to keep it as @timja underlined?

**Comment by NotMyFault** [comment]
Occasionally people file issues there because issue creation is enabled for some reason in the project.

**Comment by timja** [comment]
Probably doesn’t need issue creation as the bot just creates components atm

**Comment by MarkEWaite** [comment]
The Jira permissions for the "test" project use the same permissions set as the permissions for the "JENKINS" project.  When I removed "Create issue" permission for the "test" project, I also removed "Create issue" permission for the "JENKINS" project.

I don't think the benefit from this change is large enough to justify the complication of separating the permission differences.  I propose to close this without making a change.  @NotMyFault is that OK for you?
